src/main/java/uk/gov/ons/census/casesvc/service/RmCaseUpdatedService.java

```diff
@@ -53,7 +53,7 @@ public void processMessage(ResponseManagementEvent rme, OffsetDateTime messageTi
     caze.setSkeleton(false);
 
     Metadata eventMetadata = null;
-    if (shouldSendCaseToField(caze, rme.getEvent().getChannel())) {
+    if (shouldSendCaseToField(caze)) {
```

Comment:
what happened to the channel check? seems like we lost something here - it would be impossible for `shouldSendCaseToField` to guess the channel, no?

Reply:
have discussed - this requirement seems to have gone away